monero-outreach-docs/translations/fr/archive/we_accept_monero_fr.md

```diff
@@ -17,5 +17,5 @@ Vous pouvez télécharger un fichier prêt à être imprimé depuis [WhenYouAcce
 ### _En apprendre plus_
 
 - [Une introduction sur comment aider Monero](https://www.monerooutreach.org/stories/getting-started-helping-monero.html)
-- [Un guide essentielle pour un Marketing de type guérilla pour Monero](https://www.monerooutreach.org/guerrilla-toolkit.html)
+- [Un guide essentiel pour un Marketing de type guérilla pour Monero](https://www.monerooutreach.org/guerrilla-toolkit.html)
```

Comment:
Here change the link to the Monero Guerrilla Toolkit article: https://www.monerooutreach.org/boite-outils-marketing-guerilla.html

The name that was given is this:
[Boîte à Outils Marketing Guérilla](https://www.monerooutreach.org/boite-outils-marketing-guerilla.html)

This one has translated several months ago, let me know if the title fits or needs a change :).

Reply:
The title 'Boîte à Outils Marketing Guérilla' is a bit too much a word by word translation. It is fine to keep 'Boîte à Outils', the second part: 'Marketing Guérilla' is missing many conjunctions that cannot really be omited. So what I suggest is to use the intermediate solution: 'Boîte à Outils pour un Marketing de type Guérilla pour Monero'
pinging @Hsterofemia for his own insight on this as he already reviewed my translation and might have some opinion as well.

Reply:
I agree with your recommendation. Not an easy term to simultaneously translate correctly while making it concise and sound good.

Reply:
Thanks @Hsterofemia. @lh1008, I have updated my file according to the suggestion I made. The link is also correct, following your request.